test-cases/tests/products/h22-validate-that-rate-limit-service-is-working-as-expected.md

```diff
@@ -20,7 +20,7 @@ This test case should prove that the rate limiting Redis counter correctly incre
 
 1. Open Openshift Console in your browser
 2. Copy `oc login` command and login to your cluster in the terminal
-3. Go to `Networking` > `Routes` under `redhat-rhmi-3scale` namespace
```

Comment:
is this test case not needed for rhmi anymore?

Reply:
This test case is used to test the rate limit service, so it's valid only for RHOAM.
The test case was mentioning `redhat-rhmi` prefix, because the prefix was changed just recently to `redhat-rhoam` (after last release testing was done)

Reply:
Cool thats exactly what i was thinking, thought i would just make sure